_data/chapter/design-principles.html

```diff
@@ -0,0 +1,407 @@
+                FIG. 04: PRIMARY FACTORS OG USER EXPERIENCE
+                OWN ILLUSTRATION BASED ON IXDF - USABILITY, 2020
+            </p>
+            </div>
+        
+        <div>
+
+            
+
+        <div class="personinfo">
+            <strong>Don Norman</strong><br>
+            Director of the Design Lab at the University of California and co-founder of Nielsen Norman Group. <br><cite>(JND, N.D.)</cite>
+        </div>
+
+
+        <h3>User Experience Design</h3>
+        <p>
+            The term "user experience" was first coined by Don Norman in the early 1990s when he was working at Apple <cite>(NORMAN, 2013, P. XIII F.)</cite>. User experience comprises all aspects of a user’s interaction with a company, its services, and its products. The focus and most important goal of good user experience is to meet the user’s needs in every aspects. Close cooperation is required between different disciplines, such as engineering, marketing, graphic design, industrial design, and interface design <cite>(NORMAN & NIELSEN, N.D.)</cite>.
+        </p>
+
+        <p class="quote">
+            <q>
+            Good design is actually a lot harder to notice than poor design, in part because good designs fit our needs so well that the design is invisible, serving us without drawing attention to itself. Bad design, on the other hand, screams out its inadequacies, making itself very noticeable.
+            </q>
+            <br> 
+            <cite>NORMAN, 2013, P. XI</cite>
+        </p>
+
+        <p>
+        User experience captures everything a user experiences while using a given product, from the visuals, the haptics, and the audio to how the product feels. User experience of a product comprises three primary factors: its look, feel, and usability <cite>(IXDF - UX DESIGN, N.D.) (→ FIG. 04)</cite>.
+        </p>
+        </div>
+
+        
+            
+        <p>
+            User Experience Design requires a solid understanding of the psychology and behavior of the user. Understanding how users perceive a product allows user experience designers to iterate their concepts and better achieve the user’s goals. According to Marc Hassenzahl and Sarah Diefenbach’s paper *Well-being*, *Need Fulfillment*, and *Experience Design*, any positive experience is based on meeting psychological needs. The fulfillment of psychological needs, such as competence, relatedness, popularity, stimulation and security, contributes to the well-being of the user and leads to a pleasant user experience. In experience design, these needs are taken as guidelines and inspiration in the design process <cite>(HASSENZAHL & DIEFENBACH, 2012, P. 1)</cite>.
+        </p>
+
+        <p>
+            Some of these needs have already been discussed in Chapter 2.1, but in the following, they are considered from the perspective of emotion, while additional needs are addressed.
+        </p>
+
+        <table border-collapse: collapse>
```

Comment:
Das Attribut ist zum einen Syntaktisch inkorrekt:
```suggestion
        <table border-collapse="collapse">
```

und es wird nicht mehr empfohlen dies über ein Attribut zu lösen. Das wäre jetzt eine CSS Property